packages/@aws-cdk/aws-service-spec/build/patches/service-patches/core.ts

```diff
@@ -201,6 +201,32 @@ export namespace fp {
     );
   }
 
+  type SchemaType = "string" | "number" | "boolean" | "object" | "array" | "integer" | "null";
+
+  /**
+   * Add properties to `properties`
+   */
+  export function addPropertys<TypeName extends string = string>(
+    resource: TypeName,
+    propertyName: string,
+    propertyType: SchemaType,
+    reason: patching.Reason,
+  ): patching.Patcher<patching.JsonLens> {
+    return patchResourceAt<types.CloudFormationRegistryResource['properties']>(
+      resource,
+      '/properties',
+      reason,
+      (dbInstanceProperties) => {
+        if(!(propertyName in dbInstanceProperties)) {
+          dbInstanceProperties[propertyName] = {
+            type: propertyType
+          } as types.jsonschema.Schema;
+        }
+        return dbInstanceProperties;
+      },
+    );
```

Comment:
[nit] variable names are related to the change we were trying before, make the names more generic